README.md

```diff
@@ -44,6 +44,19 @@ without having to build it from source.
 TODO(https://github.com/mc-imperial/dredd/issues/28): add instructions
 on how to use Dredd
 
+`pi.cc` is a small example in `examples/simple` that calculates an approximation of pi. To run this example, run the following from the root of the repository:
+```
+# This will save the example file since pi.cc is modified in-place by dredd.
+cp examples/simple/pi.cc examples/simple/backup.cc
```

Comment:
Rather than having them copy pi.cc into a backup, I suggest merely including the command to mutate in place, then remarking on this (possibly noting that they can use git diff to see the changes that Dredd has made, but noting that these changes are not meant to be human readable), and then at the end of the section giving them a git checkout command to restore the file.